tests/signal.test.ts

```diff
@@ -0,0 +1,248 @@
+import { batchRequests, GraphQLClient, rawRequest, request } from '../src'
+import { setupTestServer, sleep } from './__helpers'
+import 'abort-controller/polyfill'
```

Comment:
Let's mention in the docs:

1. Supported Node versions
2. How to get abort-controller working for environments with an older Node version (apparently just this one import should do the trick?)

After that, LGTM!

Reply:
@jasonkuhrt Done in https://github.com/prisma-labs/graphql-request/pull/303/commits/19d45f89011cc833f9c2be7ddacd9e7ab226c053